nextflow_schema.json

```diff
@@ -584,6 +589,33 @@
                     "help_text": "Useful to allow reproducibility, as BUSCO datasets are frequently updated and old versions do not always remain accessible."
                 }
             }
+        },
+        "ancient_dna_assembly": {
+            "title": "Ancient DNA assembly",
+            "type": "object",
+            "description": "",
```

Comment:
Short description what it does here would be nice

Reply:
Done (Added: `Performs ancient DNA assembly validation and contig consensus sequence recalling.`)